data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReportingSystemContext.kt

```diff
@@ -11,12 +11,12 @@ import com.squareup.moshi.JsonClass
 data class GaeFeedbackReportingSystemContext(
 
   /** The package version name for the user's instance of the app. */
-  @Json(name = "version_name") val packageVersionName: String,
+  @Json(name = "platform_version") val packageVersionName: String,
   /** The package version code for the user's instance of the app. */
   @Json(name = "package_version_code") val packageVersionCode: Int,
-  /** The ISO-639 code for the country locale set on the user's device. */
-  @Json(name = "country_locale") val countryLocale: String,
-  /** The IDO-3166 code for the language locale set on the user's device. */
-  @Json(name = "language_locale") val languageLocale: String
+  /** The ISO-3166 code for the country locale set on the user's device. */
+  @Json(name = "country_locale_code") val countryLocaleCode: String,
+  /** The IDO-639 code for the language locale set on the user's device. */
```

Comment:
Were these ISO codes reversed before?

Reply:
Yup, accidentally switched them earlier. [Language codes](https://developer.android.com/reference/java/util/Locale#getISOLanguages()) are ISO-639 and [country codes](https://developer.android.com/reference/java/util/Locale#getISO3Country()) are ISO 3166.